source/code/plugin/KubernetesApiClient.rb

```diff
@@ -357,6 +373,14 @@ def getContainerResourceRequestsAndLimits(metricJSON, metricCategory, metricName
             podUid = pod["metadata"]["uid"]
           end
 
+          # For ARO, skip the pods scheduled on to master or infra nodes to ingest
+          if isAROCluster && !pod["spec"].nil? && !pod["spec"]["nodeName"].nil? &&
+            ( pod["spec"]["nodeName"].downcase.include?("infra-") ||
```

Comment:
please use starts_with instead if include. Include checks for substrings. and if a node has 'infra-' or 'master-' anywhere in its name it will be excluded when u use include